terraform/panther_gcs_transport_type_infra/production.tfvars

```diff
@@ -1,10 +1,18 @@
-project_id                           = "project-id"
-bucket_name                          = "bucket-name"
-topic_name                           = "topic-name"
-subscription_id                      = "subscription-id"
-gcp_region                           = "gcp-region"
-gcp_zone                             = "gcp-zone"
-gcs_bucket_location                  = "gcs-bucket-location"
-gcs_bucket_prefixes                  = ["logs/", "audit-logs"]
+# Basic Info
+project_id = "project-id" # ID (name) of your GCP project
+gcp_region = "gcp-region" # GCP Region that your infrastructure will live
+gcp_zone   = "gcp-zone"   # GCP Availability Zone that your infrastructure will live
+
+# Panther will pull data from this GCS bucket by way of Pub/Sub Notifications
+# If your bucket already exists you would need to remove the bucket_name and the accompanying code in the TF module
+bucket_name         = "bucket-name"
+topic_name          = "topic-name"
+subscription_id     = "subscription-id"
+gcs_bucket_location = "gcs-bucket-location"
+
+# Prefixes that notifications will be setup for. Data will need to be added within thoses prefixes
+gcs_bucket_prefixes = ["logs/", "audit-logs"]
```

Comment:
This is interesting... are the customers meant to replace these prefixes with their own? If yes, we should call that out explicitly e.g. "Replace the following prefixes with the prefixes you want Panther to consume data from" or something

Reply:
That's a good point. I'm in favor of abstract values that clearly correspond to example values. So for example `example-prefix-1/`. These very specific values could be mistaken as Panther restrictions, meaning that a user could assume that we only read from these prefixes?

Reply:
Nice thinking. Changed.